apps/emqx_durable_storage/src/emqx_ds_bitmask_keymapper.erl

```diff
@@ -343,14 +348,15 @@ make_filter(
         [],
         Schema
     ),
-    Ranges = array:from_list(lists:reverse(Filter)),
+    Ranges = list_to_tuple(lists:reverse(Filter)),
```

Comment:
This seems to have changed the type of `#filter.bitsource_ranges` from `array:array(_)` to `tuple(_)`, but `emqx_ds_bitmask.hrl` wasn't updated to reflect this change.